integration/presto/src/main/java/org/apache/carbondata/presto/constants/CarbonCommonConstants.java

```diff
@@ -0,0 +1,1319 @@
+/*
```

Comment:
please only keep these CommonConstants which be used by presto module, don't need to copy all.